README.md

```diff
@@ -0,0 +1,147 @@
+_This is draft email to the lightning-dev mailing list. If all goes well maybe it can also be a place iterate on the design._
+
+# Background
+
+As presently specified, the two parties in a lightning channel are assigned different commitment transactions.
+This is logically necessary for the protocol to identify which party broadcasted the commitment transaction and potentially punish them if the other party provides provides proof it has been revoked (i.e. knows the revocation key).
+
+Wouldn't it be nice if we could identify the broadcasting party without assigning them different transactions?
+Riard first considered this problem in [8] and proposed that you could identify the party using _witness asymmetry_ i.e. both parties broadcast the same transaction but have different witnesses. 
+Unfortunately, the solution they propose is rather convoluted.
+
+More recently in [1], Aumayr et al. introduced a much more elegant witness asymmetric solution using adaptor signatures.
+Instead of being assigned different transactions, the parties are assigned different adaptor signatures as witnesses for the _same_ transaction.
+The adaptor signatures force the party broadcasting a commitment transaction to reveal a "publishing secret" to the other party.
+The honest party can then use this publishing secret along with knowledge of the usual revocation secret to punish the malicious party for broadcasting an old commitment transaction.
+
+In the paper this idea is combined with "punish-then-split" mechanism similar to the original eltoo[8] proposal and therefore it unfortunately inherits the issue with staggering time-locks.
+BOLT 3 [4] notes this problem as design motivation:
+
+> The reason for the separate transaction stage for HTLC outputs is so that HTLCs can timeout or be fulfilled even though they are within the to_self_delay delay. Otherwise, the required minimum timeout on HTLCs is lengthened by this delay, causing longer timeouts for HTLCs traversing the network.
+
+More in depth commentary by Towns can be found in [2] and they propose a solution to the eltoo proposal in [3].
+
+In short, the problem is that it creates a risk for the party that needs to fulfill an HTLC with the secret in time.
+The only known way of accounting for this risk is to increase the difference between the time-locks on each hop.
+There could be situations where this trade off makes sense but it seems like it undesirable for a general purpose payment channel network.
```

Comment:
"as it would make any HTLC-targeting attack easier" like flood & loot, pinning, time-dilation.

Recent trend has been to increase them, https://github.com/lightningnetwork/lightning-rfc/pull/785

Reply:
I'm not sure I can justify that claim. If you account for the relative time-lock by extending the absolute time-lock doesn't that address this (I mean in so far as it's not worse than the current state of a affairs)?

I have a feeling you are right in practice but I don't see the mechanics of it. The analysis of these attacks is certainly more difficult with the punish-then-settle model.
It might be time to do some math and stop using feelings though!